### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][NVDA][MAS4.2.1][IntelliJ][Wipro]: NVDA is not announcing appropriatey the Static Text present in sign in dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1486]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA
3. Tab to Tools menu, point to "Azure", and then select "Azure Sign In".
4.  In the Azure Sign In Dialog, try to access the static text then observe the NVDA Behavior

**Expected Result:**
**NVDA Behavior:** NVDA Should announce the "You will manually sign in using your Azure credentials as needed." for the text present under Interactive Radio button.
NVDA Should announce "An authentication file with credentials for an Azure Active Directory service principal will be used for automated sign ins." for text presenter Under Automated Radio button

**Actual Result:**
**NVDA Behavior**: NVDA is announcing as edit text/plain  credentials as needed for text present under Interactive radio button
NVDA is announcing as edit text/plain will be used for automated signins for text present under Automotive radio button
**Jaws Behavior:** Jaws is announcing as expected
read only edit
You will manually sign in using your Azure credentials as needed.
text/plain
Insert f1 Help.
Automated radio button not checked
1 of 2
An authentication file with credentials for an Azure Active Directory service principal will be used for automated sign ins.
text/plain

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![4f1825cf](https://user-images.githubusercontent.com/22956130/36134711-a4788a34-10ac-11e8-9b12-ccbcc1dcd0b0.jpg)
![nvda signin](https://user-images.githubusercontent.com/22956130/36134712-a4aa866a-10ac-11e8-8b43-0ae48eb4d60e.jpg)
![sign in-static](https://user-images.githubusercontent.com/22956130/36134713-a4d734c6-10ac-11e8-963b-698e5b6e2959.jpg)

**Comment by Eskibear** [comment]
under further investigation

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749958
